sig-service-catalog/charter.md

```diff
@@ -0,0 +1,131 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI).
+It allows application developers the ability to provision and consume cloud services natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
+- Provide a UX for Kubernetes users that is consistent with both the OSB API specification and traditional Kubernetes user interactions.
+- Align with the [OSBAPI specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md) as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
+
+### Out of scope
+
+The following, non-exhaustive, items are out of scope:
+- Operation of OSBAPI Service Brokers.
+
+## Roles
+
+- Maintainers
+  - Responsible for approving, and reviewing, pull requests.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by a chair, to accepted via lazy two-thirds
+    resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    to accepted via lazy two-thirds resolution amongst the chairs.
+
+- Chairs
+  - All maintainer’s roles.
+  - Responsible for project administration activities within the SIG and are non-technical
+    in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback to a 2/3 majority vote (lazy two-thirds resolution).
+    This process is used for all decisions, such as changing chairs/maintainers or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time. Before stepping down, the chair
+    may propose and vote on their replacement via lazy two-thirds resolution amongst the chairs.
```

Comment:
This implies there is a fixed number of chair seats, which I do not think is the case - despite our attempt to have a number of key companies represented. So, we should either state 'there will be 4 chairs', then this sentence about someone picking their replacement makes sense, or we should not talk about picking replacements at all and let things just happen via the normal nomination process. I don't think I was clear on this point in my previous comment.

Reply:
I don't think the "their replacement" implies a fixed number.

Reply:
The intent was to allow someone who is stepping down to participate in nominating and voting on people, i.e. they aren't lame ducks who as soon as they hint that they are on the way out, can't participate.

No number of replacements was meant to be implied.

How about "Chairs may decide to step down at any time. Before stepping down, the chair may continue to nominate new chairs and participate in the vote."

Reply:
"their replacement" implies a vacancy, which implies there's an assumed number of chairs rather then people become a chair based on something else.  Or put another way, why are we linking stepping down with adding new ones at all? To me they are totally independent actions.

Reply:
I like @carolynvs - I'm basing the following on that, making modifications to help signify that there are not a set number of chairs:

```
Chairs may decide to step down at any time. Before officially stepping down, the chair may decide to, but are not required to, nominate one or more new chairs and participate in votes for them. 
```

Reply:
@duglin What do you think of the new text that I suggested? It doesn't use the term replacement.

Reply:
@arschles its better but its still tortured. I'm not sure why there's a desire to link the two actions - perhaps I'm just missing something.  Why not just say `Chairs may decide to step down at any time. ` and that's it. Why do we need a 2nd sentence at all? the previous bullet talks about how to add chairs - what else is needed? What is the idea trying to be conveyed?  Why should one chair stepping down cause a new one to be added.... unless we're trying to find a replacement w/o calling it a "replacement" ;-)  If a chair that is stepping down really want to be involved in finding "another chair", then do the nominations before they step down.

Reply:
A few questions in here; I'll try to boil it down. The text acknowledges and encodes the fact that a chair can participate in finding someone to take their place, if they want someone to take their place. That's the idea to be conveyed here.

Reply:
> The text acknowledges and encodes the fact that a chair can participate in finding someone

The previous bullet already says that.

> to take their place

This is the rub. We didn't select this "other person" just because we think they should be a chair on their own, but instead its only because someone else is stepping down. This implies a link between the two actions - which is fine, if that's what we want, but then define it. Is it because there's a fixed number of seats? One from each key company? Does it imply we can't have more than one co-chair from a company? I'm not trying to be a pain, but there's an implied rule behind the proposed text and I'm looking for clarity on what it is.

Personally, I think the only thing we should state is that there can't be more than one co-chair from the same company. But just because someone from company X is stepping down doesn't mean we MUST find a replacement for them because there may not be someone else from that company who is active enough to justify it. And if we want to pick someone from another company, then we should have picked them regardless of the vacancy otherwise I would question if they were really worthy - meaning it shouldn't have been based on one person stepping down. And if it is due to someone stepping down, then that (again) implies some kind of fixed number of seats... so let's define that.

Reply:
Ok, I acknowledge that the previous bullets cover this bullet. I'm fine leaving it out. As you know, I agree with you that the number of chairs should not be limited at this point

Reply:
updated